domains/meowcat.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "meowcat767"
+    },
+    "records": {
+        "A": "[185.199.108.153]"
+        "A": "[185.199.109.153]"
+        "A": "[185.199.110.153]"
+        "A": "[185.199.111.153]"
+    }
```

Comment:
This is not valid JSON. All IPs in A records should be placed into a single array, with a single object property. Re-open when this PR is fixed.